balloon.py

```diff
@@ -209,13 +209,12 @@ def balloon_m(
                     delta=delta,
                 )
             )
-        for future in concurrent.futures.as_completed(futures):
-            result = future.result()
-
-            if len(output) == 0:
-                output = result
-            else:
-                output = bytes([_a ^ _b for _a, _b in zip(output, result)])
+        completed_futures = concurrent.futures.as_completed(futures)
+        for future in completed_futures:
+            output = future.result()
+            break
```

Comment:
Can't we use `next()` here instead?

Reply:
I can put a `next(completed_futures, "")` to avoid the need to catch a `StopIteration` exception.

Reply:
Sure, that would be nice!